task_1/dmitriy.osipov/src/main/java/com/task/model/Blob.java

```diff
@@ -0,0 +1,66 @@
+package com.task.model;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Blob implements IElementContent {
```

Comment:
Don't use **I** prefix for interfaces. You can get to know that it is an interface by the **implements** keyword.